svtools/bedpetovcfconverter.py

```diff
@@ -0,0 +1,111 @@
+from svtools.vcf.variant import Variant
+import re
+
+class BedpeToVcfConverter(object):
+    '''
+    This is a class to take Bedpe objects and convert them to VCF variant object(s)
+    '''
+
+    def __init__(self, vcf):
+        '''
+        Initialize a new converter with a reference to Vcf object
+        '''
+        self.vcf_header = vcf
+
+    @staticmethod
+    def adjust_by_cipos(bedpe):
+        '''
+        Undo adjustments to BEDPE coordinates for the VCF POS based on confidence intervals
+        '''
+        position = bedpe.s1
+        if bedpe.o1 == '-' and bedpe.svtype == 'BND':
+            position += 1   #undo left adjust based on strandedness
+        if 'CIPOS=' in bedpe.info1:
+            cipos = re.split('=|,', ''.join(filter(lambda x: 'CIPOS=' in x, bedpe.info1.split(';'))))
+            position -= int(cipos[1])
+        return position
+
+    @staticmethod
+    def adjust_by_ciend(bedpe):
+        '''
+        Undo adjustments to BEDPE coordinates for the VCF END field based on confidence intervals
+        '''
+        end = bedpe.s2
+        if bedpe.o2 == '-' and bedpe.svtype == 'BND':
+            end += 1
+    
+        if 'CIEND=' in bedpe.info1:     
+            ciend = re.split('=|,', ''.join(filter(lambda x: 'CIEND=' in x, bedpe.info1.split(';'))))
+            end -= int(ciend[1])
+        return end
+
+    @staticmethod
+    def determine_sep(o2):
+        '''
+        Given the orientation of the other breakend, determine separator for VCF alt
+        '''
+        if o2 == '+':
+            return ']'
+        else:
+            return '['
+
+    @staticmethod
+    def determine_flanks(o1):
+        '''
+        Given the orientation of the breakend, return proper flanking sequence strings
+        '''
+        if o1 == '+':
+            return ('N', '')
+        else:
+            return ('', 'N')
+
+    def bnd_alt_string(self, o1, o2, chrom, pos):
+        '''
+        Given a Bedpe object generate the correct alt string for a BND VCF entry
+        '''
+        alt_string = '{3}{0}{1}:{2}{0}{4}'
```

Comment:
this is a staticmethod?
`alt_string = '{3}{0}{1}:{2}{0}{4}'`
it looks like just a string of placeholders